servers/annotations-api/src/models/HighlightsModel.ts

```diff
@@ -0,0 +1,95 @@
+import { ForbiddenError } from '@pocket-tools/apollo-utils';
+import { HighlightsDataService } from '../dataservices/highlights';
+import { NotesDataService } from '../dataservices/notes';
+import {
+  BatchWriteHighlightsInput,
+  BatchWriteHighlightsResult,
+  Highlight,
+  CreateHighlightInput,
+  HighlightNote,
+  UpdateHighlightInput,
+  SavedItemAnnotations,
+  CreateHighlightByUrlInput,
+} from '../__generated__/resolvers-types';
+import { ParserAPI } from '../dataservices/parserApi';
+
+export class HighlightsModel {
```

Comment:
this is just moving the code from `resolvers.ts` into a model for better visibility